src/index.ts

```diff
@@ -110,6 +110,16 @@ export interface IClientConfig {
   persistentStore?: IAsyncStore<Flag>;
 }
 
+export interface IClientConfigSync {
+  flagsConfiguration: Record<string, Flag>;
+
+  assignmentLogger?: IAssignmentLogger;
+
+  isObfuscated?: boolean;
+
+  throwOnFailedInitialization?: boolean;
+}
+
```

Comment:
These are the possible configuration options for calling the `initSync` method - note that no SDK key is provided as it will be operating in off-line mode.

Assuming that on the server the user stringified the configurations to a JSON string, they will need to parse it back into the object using the types we export in the SDK.